Lib/glyphsLib/types.py

```diff
@@ -405,3 +409,46 @@ def plistValue(self, format_version=2):
         # TODO write hex bytes in chunks and split over multiple lines
         # for better readability, like the fonttools xmlWriter does
         return "<%s>" % binascii.hexlify(self).decode()
+
+
+class Pointer(ValueType):
```

Comment:
Is this class named like that in Glyphs.app's python API as well? just curious

Reply:
Mmm, no. I just came up with something that would capture the idea that we are pointing at where to attach. I do not think there is a dedicated `GSSomething` for this in the API.

Reply:
ah. Well, "Pointer" has quite a baggage... Can we named it something else? It looks like this represents a variable-length sequence of int or str, am I right? Maybe call it `Sequence`. Or if it's only ever used for GSHints, maybe give it a  name that describes what it's used for..

Reply:
what type do you get for an hint's `origin` or `target` if you print from within Glyphs.app macro window?

Reply:
Those are an objc class called "[NSIndexPath](https://developer.apple.com/documentation/foundation/nsindexpath?language=objc)". It is a list of indexes. They are used to identify things in a tree of objects (like selections in an outlineview.

Reply:
OK, I can rename it to IndexPath.

Reply:
I am reading the documentation. It does not seem to match the description of `NSIndexPath`. According to

https://github.com/schriftgestalt/GlyphsSDK/blob/Glyphs3/GlyphsFileFormat/GlyphsFileFormatv3.md

the tuple can occasionally contain only one element in which case it is a string equal to `lsb` or `rsb` for `origin` or `up` and `down` for `target`. But conceptually, those for strings are kind of indices.

I have renamed to `Index` and added documentation. What do you think?

Reply:
That it is implemented as an NSIndexPath is not important. 

And in the python (and objc) API, it is always an index path, with some special indexes denoting the "lsb", "rsb" … values. Only in the written file it is converted to those strings.

Reply:
I agree that it is not important and thank you for clarifying. Then it is adequate, but let me rename to IndexPath to be crystal clear.